source/updateCheck.py

```diff
@@ -482,12 +482,12 @@ def __init__(self, parent, destPath, version, apiVersion, backCompatTo):
 		self.apiVersion = apiVersion
 		self.backCompatTo = backCompatTo
 		self.storeUpdatesDirWritable = os.path.isdir(storeUpdatesDir) and os.access(storeUpdatesDir, os.W_OK)
-		# Translators: The title of the dialog asking the user to Install an NVDA update.
+		# Translators: The title of the dialog asking the user to apply an NVDA update.
 		super().__init__(parent, title=_("NVDA Update"))
 		mainSizer = wx.BoxSizer(wx.VERTICAL)
 		sHelper = guiHelper.BoxSizerHelper(self, orientation=wx.VERTICAL)
-		# Translators: A message indicating that an updated version of NVDA is ready to be installed.
-		message = _("NVDA version {version} is ready to be installed.\n").format(version=version)
+		# Translators: A message indicating that an updated version of NVDA is ready to apply.

```

Comment:
```suggestion
		# Translators: A message indicating that a new version of NVDA is ready to update.

```

Reply:
I agree with @CyrilleB79 that using "install" in some places and other verbiage in other places is confusing and poor practice. Personally I think that using "update" in contexts like `"NVDA version {version} has been downloaded and is ready to update."` feels agrammatical, and almost feels like you're saying the newly downloaded version needs to be updated. Perhaps rewording to something like `"Update to NVDA version {version} has been downloaded and is ready to apply."` would be better.

Reply:
That sounds good. Let's go with that.